index.html

```diff
@@ -5,14 +5,104 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Spotify Clone</title>
-    <!-- don't forget to link your styles -->
+    <link rel="stylesheet" href="styles/style.css">
+    
   </head>
   <body>
-    Premium Discover Help Download Music for everyone. Spotify is now free on mobile, tablet and computer. Listen to the
-    right music, wherever you are. What’s on Spotify? Millions of Songs There are millions of songs on Spotify HD Music
-    Listen to music as if you were listening live Stream Everywhere Stream music on your smartphone, tablet or computer
-    It’s as yeezy as Kanye West. Search Know what you want to listen to? Just search and hit play. Browse Check out the
-    latest charts, brand new releases and great playlists for right now. Discover Enjoy new music every Monday with your
-    own personal playlist. Or sit back and enjoy Radio.
+    <header>
+      <div class="head">
+        <div>
+          <img src="images/spotify-logo.png" alt="Spotify Logo">
+        </div>
+        <nav>
+          <ul>
+            <li><a href="item1">Premium</a></li>
+            <li><a href="item2">Discover</a></li>
+            <li><a href="item3">Help</a></li>
+            <li><a href="item4">Download</a></li>
+          </ul>
+        </nav>
+      </div>
+      <div class="landing">
+        <article>
+          <h1>
+            Music for everyone. 
+          </h1>
+          <p>
+            Spotify is now free on mobile, tablet and computer. Listen to the
+            right music, wherever you are.
+          </p>
+        </article>
+      </div>
+    </header>
+
+    <section class="middle">
+      <h2>
+        What’s on Spotify?
+      </h2>
+      <hr>
+      <div class="icons">
+        <box>
+          <icon>
+            <img src="/images/music-icon.png">
+          </icon>
+          <h3>
+            Millions of Songs
+          </h3>
+          <p>
+            There are millions of songs on Spotify
+          </p>
+        </box>
+        <box>
+          <icon>
+            <img src="/images/high-quality-icon.png">
+          </icon>
+          <h3>
+            HD Music
+          </h3>
+          <p>
+            Listen to music as if you were listening live
+          </p>
+        </box>
+        <box>
+          <icon>
+            <img src="/images/devices-icon.png">
+          </icon>
+          <h3>
+            Stream Everywhere
+          </h3>
+          <p>
+            Stream music on your smartphone, tablet or computer
+          </p>
+        </box>
+      </div>
+    </section>
+    
+    <section class="kanye">
+      <img src="/images/spotify-icon-white.png"></img>
```

Comment:
Nice work! in here tho i would suggest to put the text in a div and then use flex box with the div, the logo and the image of kanye west, If you have questions regarding this let me know